### PR TITLE
Angular form submission and restyling

### PR DESCRIPTION
**Pull request by vd-dennis**
feat(**): Added angular to take care of the form submission to the API and automated the dev process with Grunt. 
refactor(index/sass): Changed the layout of the form and restyled needed elements

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          ],
+        dest: 'public/js/main.js'
+      },
+      
+    },
+    uglify: {
+      build: {
+        files: {
+          'public/js/main.min.js': [
+            'node_modules/angular/angular.js',
```
Comment (glemmaPaul) [inline Gruntfile.js:37]: da werkt ook haa

File: public/index.html [hunk 2]
```diff
@@ -33,17 +33,119 @@
...
+                        ng-show="signUpForm.$submitted && signUpForm.firstName.$error.required">
+
+                        You first name is required.
+                    </span>
+
+                    <span 
+                        class="form-warning ng-cloak" 
+                        ng-show="signUpForm.$submitted && signUpForm.lastName.$error.required">
+
+                        You first last is required.
```
Comment (glemmaPaul) [inline public/index.html]: right I see my first last 🃏

Comment (glemmaPaul) [reply public/index.html]: hehe

File: public/index.html [hunk 3]
```diff
@@ -33,17 +33,119 @@
...
+                        ng-show="signUpForm.$submitted && signUpForm.politicalSide.$error.required">
+
+                        You need to select a political preference
+                    </span>
+
+                    <span 
+                        class="form-warning last ng-cloak" 
+                        ng-show="signUpForm.$submitted && signUpForm.phoneNumber.$error.required">
+
+                        You phone number is not correct (example).
```
Comment (glemmaPaul) [inline public/index.html]: example: +1 917 123-4567

**Review by glemmaPaul — APPROVED** [review]

File: public/index.html (hunk 2 above)
Comment (vd-dennis) [reply public/index.html]: whoops 😛

**Review by glemmaPaul — APPROVED** [review]
🚀

**Pull request merged**